public/styles/styles.less

```diff
@@ -10,9 +10,11 @@ body {
 }
 
 .wrapper {
-    background-image: url("https://images.unsplash.com/photo-1522223142907-0fbfb9c571c1?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=b7a69d28bf8e5030872cb254ba06efbd&auto=format&fit=crop&w=2500&q=80");
+    background: url(https://images.unsplash.com/photo-1522223142907-0fbfb9c571c1?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=b7a69d28bf8e5030872cb254ba06efbd&auto=format&fit=crop&w=2500&q=80)no-repeat center center fixed; 
```

Comment:
Properly name styles. This "background" needs to be called "background-image" in order to work.